src/css_parser.rs

```diff
@@ -2928,6 +3001,26 @@ mod css_tests {
         assert_eq!(parse_css_color("#EEE"), Ok(ColorU { r: 238, g: 238, b: 238, a: 255 }));
     }
 
+    #[test]
+    fn test_parse_css_color_4() {
+        assert_eq!(parse_css_color("rgb(192, 14, 12)"), Ok(ColorU { r: 192, g: 14, b: 12, a: 255 }));
```

Comment:
There are some other inputs that aren't tested:

- `rgb(192, 14, 12` - leaving off the trailing brace should result in an error other than just "couldn't parse" (see the `image()` and `linear-gradient()` functions where this check for the second brace exists). This is just to give more descriptive error messages.
- `rgb(` or `rgb()` - would just be interesting to how the parser reacts to null input - often you discover things such as accidental subtract-minus-one-underflows errors